robot.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you pretty format the xml, single line files are not great for git for when we later change things.

eg:

cat robot.xml | xmllint --format - 

also if at all possible I try to avoid too much stuff ended up dumps into the root directory, and these files won't be directly used by the plugin, only ever downloaded and installed. So please make a new dir  'roles' and tuck them into there